roles/ceph-rgw-loadbalancer/meta/main.yml

```diff
@@ -0,0 +1,13 @@
+---
+galaxy_info:
+  author: Gui Hecheng
+  description: Config HAProxy & Keepalived
+  license: Apache
+  min_ansible_version: 2.4
```

Comment:
```suggestion
  min_ansible_version: 2.8
```

Reply:
@guits as I understand, version 2.4 is too low for some ansible features ?

Reply:
stable-4.0 (which is the only branch where this patch will be backported) supports ansible 2.8 only

Reply:
OK